spyql/cli.py

```diff
@@ -1,6 +1,6 @@
+import spyql
```

Comment:
❓ I may be missing something super obvious, but why did you add this line? At a first glance doesn't look like it's needed.

Reply:
I noticed at line 79,

```python
@click.version_option(version=spyql.__version__)
```

so I think this imports is needed.